README.md

```diff
@@ -29,47 +34,106 @@ Just select _Preferences_ from the _Edit_ menu to enable (or disable) the explor
 
+## Development Environment
+
+To set up a _free_ Windows development environment that works with Lessmsi:
+
+1. Install [VMWare Fusion Player](https://www.vmware.com/products/fusion.html) (using free "Player" version, only needed if developing on MacOS)
+2. Download a Virtual Machine for Windows development that Microsoft provides at https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/. These VMs include virtual machine with the latest versions of Windows, the developer tools, SDKs, and samples ready to go.
+3. Install chocolatey as described at https://chocolatey.org/install
+4. Install some things using the "Command Prompt" (`cmd` rather than "Power Shell"/`ps`) and `winget` here):
 
-# Donate to Support Open Source Contributors of lessmsi #
-You can **donate** in two ways: 
-* [Donate at Tip4Commit](https://tip4commit.com/github/activescott/lessmsi) (Bitcoin only) to fund a general fund. Each new commit to this repository receives a percentage of the available balance.
-* [Donate at IssueHunt](https://issuehunt.io/r/activescott/lessmsi) to deposit your donation as a "bounty" against a specific issue or feature request. When a contributor resolves the issue they will earn the deposit for that issue.
+```sh
+# install git for windows (this also installs "bash" via "Git Bash")
+$ winget install git.git
 
+# install chocolatey (this is required to run the MSBuild script and create the chocolatey package that is deployed by CI)
+$ winget install chocolatey
+
+# NOTE: you'll need to close this window and open a new one to get the cpack and git on the path
+```
+
+Then you can switch to bash by typing "bash" in the start menu and selecting "Git Bash" and follow these steps:
+
+```sh
+# in Git Bash..., generate and ssh key for github:
+$ ssh-keygen
+# follow prompts...
+
+# Print *public* ssh key to console, and register the printed value at github at https://github.com/settings/ssh/new by copying the key printed from the prior step into that box
+$ cat ~/.ssh/id_ed25519.pub
+
+# create directories for code and clone the repo:
+$ mkdir /c/src
+$ cd /c/src
+$ git clone git@github.com:activescott/lessmsi.git
+```
+
+Now I switch back to a "Developer Command Prompt for Visual Studio" (i.e. `cmd` with helpful PATH) to do a build since it is more CI-like:
+
+```sh
+$ cd \src\lessmsi\src
+$ .\build.bat
+
+# I usually start with 0.0.1 when prompted for a version number...
+
+```
+
+## Supported Windows Versions
+
+The latest versions of Lessmsi should support at least the oldest version of Windows that Microsoft still supports according to https://learn.microsoft.com/en-us/windows/release-health/supported-versions-windows-client
+That means we need to also target a .NET Framework version that also supports the oldest version Windows that Microsoft still supports which can be found at https://en.wikipedia.org/wiki/.NET_Framework_version_history
```

Comment:
Clarify here that we should target a version of .NET Framework that is included in the oldest version of windows that is still supported. For example as of 2024, Windows 10 is the oldest version of Windows still supported and according to the Wikipedia article, .NET Framework 4.8 is included in Windows 10 (some reasonably current update to Windows 10), so targeting .NET Framework 4.8 is ideal since any reasonably up-to-date version of Windows 10 (the oldest supported version of Windows) will have .NET Framework 4.8.